docs/development/doc_howto.rst

```diff
@@ -11,7 +11,7 @@ Documentation building
 
 Generating the HTML docs for Gammapy is straight-forward::
 
-    make docs-all
+    tox -e build_docs
```

Comment:
technically `make docs-sphinx` works (docs-all is no longer required since notebooks have been replaced by python scripts). 
So I would keep it here and propose `tox -e build_docs` as another way to build the docs.

Reply:
But then we should put `make docs-sphinx`instead of `make docs-all`right ?